apps/cowswap-frontend/public/halloween.css

```diff
@@ -0,0 +1,14 @@
+@keyframes blinker {
+  50% {
+    background: black;
+  }
+
+  100% {
+    background: white;
+  }
+}
+
+body.lightning {
+  filter: invert(1) grayscale(1);
+  animation: blinker 333ms linear infinite;
+}
```

Comment:
I know this is a pure revert, but there were some improvements, like removing this CSS file